bochs/cpu/cpu.h

```diff
@@ -954,6 +954,21 @@ class BOCHSAPI BX_CPU_C : public logfunctions {
   Bit32u wr_pkey[16];
 #endif
 
+#if BX_SUPPORT_UINTR && BX_SUPPORT_X86_64
```

Comment:
Is there any need to check ``BX_SUPPORT_X86_64`` here? I thought ``BX_SUPPORT_UINTR`` can't be enabled without it anyway.

Reply:
I started with attempt to avoid configure time restrictions but recently changed my mind.
Will cleanup all these && BX_SUPPORT_X86_64 later

Reply:
@stlintel 
Will you include this cleanup into the same PR or it'll be really later?

Reply:
A bit later. Now I am trying to make some assembly tests to see that it works as expected.